index.html

```diff
@@ -14,31 +14,23 @@
   </head>
 
   <body>
+    <section>
     <!--header-->
     <header class="header">
-<<<<<<< HEAD
-      <img class="logo" src="./logo.svg" alt="LOGO" />
+      
+      <div class="logo">
+        <a href="index.html"><img  src="./logo.svg" alt="LOGO" /></a>
+      </div>
```

Comment:
- [ ] As the previous reviewer mentioned please add the correct image path to make it work online.
[Hint]: You can do it like that `<img  src="./images/logo.svg" alt="LOGO" />` 

- [ ] [Optional][Highly recommended] You don't have the need to wrap the logo link with a `<div>` tag so please remove this wrapping div for semantic reasons.